### PR TITLE
added: Add Chinese regional text display content

### PR DESCRIPTION
**Pull request by huifer**
Thank you for submitting a pull request!

## Description

I've added Chinese regional text information to this project

## Type of Pull Request

- [x] Adding a feature
- [ ] Fixing a bug
- [ ] Maintaining documents
- [ ] Others ()

## Verify the followings

- [x] Code is up-to-date with the `main` branch
- [x] No lint errors after `npm run lint`
- [x] Make sure all the exsiting features working well

Refer to [CONTRIBUTING.MD](https://github.com/watergis/mapbox-gl-export/tree/master/.github/CONTRIBUTING.md) for more details.

### CODE REVIEW

**Comment by huifer** [comment]
@JinIgarashi   Please help assist to see if the relevant content is reasonable

File: README.md
Comment (JinIgarashi) [inline README.md:125]: Can we use `-` for language code instead of `_`? I think `-` is generally used to express a language code.

Also, do we need to add `CN` in the last of code? If there are any differences in translations by regions like China, Hong Kong , Taiwan..., maybe `CN` is necessary. But we may not need it in this plugin. I think shorter code might be better

File: lib/export-control.ts
Comment (JinIgarashi) [inline lib/export-control.ts:19]: As I commented previously, we can use `-` and we may not need regional name like `CN` in a language code.

File: lib/local/index.ts
Comment (JinIgarashi) [inline lib/local/index.ts:8]: This one, too. See my previous comment

**Review by JinIgarashi — CHANGES_REQUESTED** [review]
@huifer Thank you for the pull request! This translation for Chinese languages is awesome! I just commented few minor things which you can change it before merging to master branch.

**Comment by JinIgarashi** [comment]
@huifer I am happy to merge this PR to master branch if you can rename files and variables as follows.

-  for the language parameter
  - `zh_Hans_CN` > `zh-Hans`
  - `zh_Hant_CN` > `zh-Hant`
- for the variable name
  - `zhHansCN` > `zhHans`
  - `zhHantCN` > `zhHant`
- for the local language files' name
  - `zh_Hans_CN.ts` > `zhHans.ts`
  - `zh_Hant_CN.ts` > `zhHant.ts`

Thanks

File: README.md
Comment (huifer) [reply README.md:125]: OK

**Comment by huifer** [comment]
Let's confirm some final issues now. How would you suggest changing the name for the following content


zh_Hans_CN -> zh-Hans
zh_Hant_TW -> zh-Hant-TW
zh_Hant_HK -> zh-Hant-HK

**Comment by JinIgarashi** [comment]
@huifer Thanks for changing. looks great

**Comment by JinIgarashi** [comment]
@huifer [v2.0.2](https://github.com/watergis/mapbox-gl-export/releases/tag/v2.0.2) was now released. Thank you for your contribution!

**Pull request merged**
